tests/test_trainer.py

```diff
@@ -411,5 +412,103 @@ class CurrentTestModel(
     trainer.test()
 
 
+def test_dataloaders_passed_to_fit(tmpdir):
+    """Verify that dataloaders can be passed to fit"""
+    tutils.reset_seed()
+
+    class CurrentTestModel(
```

Comment:
`LocalTestModel` ?

Reply:
Since all the other test call it `CurrentTestModel`, I think it should remain as such